app/code/Magento/Catalog/Controller/Adminhtml/Category/Add.php

```diff
@@ -6,31 +6,38 @@
  */
 namespace Magento\Catalog\Controller\Adminhtml\Category;
 
-use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Model\View\Result\Page;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Catalog\Controller\Adminhtml\Category;
+use Magento\Backend\Model\View\Result\ForwardFactory;
+use Magento\Framework\App\Action\HttpGetActionInterface;
 
 /**
  * Class Add Category
  *
  * @package Magento\Catalog\Controller\Adminhtml\Category
  */
-class Add extends \Magento\Catalog\Controller\Adminhtml\Category implements HttpGetActionInterface
+class Add extends Category implements HttpGetActionInterface
 {
     /**
      * Forward factory for result
      *
-     * @var \Magento\Backend\Model\View\Result\ForwardFactory
+     * @var ForwardFactory
```

Comment:
```suggestion
     * @deprecated
     * @var ForwardFactory
```

Reply:
Please squash it with your changes ofc

Reply:
Please add comment why it's deprecated.

Reply:
Just because it's not used? I'm not even sure it was ever used

Reply:
just wanted to add description why it was deprecated, like this
```suggestion
     * @deprecated not used anymore
     * @see $this->resultFactory->create()
     * @var ForwardFactory
```

Reply:
Done, thank you.